OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -56,6 +57,10 @@ public override Activity Tick(Actor self)
 					inner = ActivityUtils.SequenceActivities(new FlyTimed(ticksUntilTurn, self), new Fly(self, target), new FlyTimed(ticksUntilTurn, self));
 				else
 					inner = ActivityUtils.SequenceActivities(new Fly(self, target), new FlyTimed(ticksUntilTurn, self));
+
+				// This needs to be done in this round-about way because TakeOff doesn't behave as expected when it doesn't have a NextActivity.
```

Comment:
Can we prepend a `// HACK: ` to this?  We really should fix TakeOff in a future PR.